### PR TITLE
Task #68988: Create/edit checklist item

### PR DESCRIPTION
**Pull request by betty2310**
## Related Tickets
- [ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/68988)

## WHAT (optional)
- Tạo và sửa item checklist sức khoẻ dự án

## Evidence (Screenshot or Video)

CleanShot 2023-09-25 at 14 15 00

### CODE REVIEW

**Comment by ghost** [comment]
oke

**Comment by hadtt-3500** [comment]
oke

**Comment by betty2310** [comment]
ready

File: config/routes.rb [hunk 1]
```diff
@@ -20,6 +20,6 @@
     resources :project_features
     resources :health, only: %i(new create edit update)
     resources :value_resources, only: :index
-    resources :health_items, only: :index
+    resources :health_items, only: %i(index new create edit update)
```
Comment (ngocvt-0484) [inline config/routes.rb]:
```suggestion
    resources :health_items, except: :destroy
```

**Comment by betty2310** [comment]
ready

**Pull request merged**